.github/workflows/ci.yaml

```diff
@@ -0,0 +1,46 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  tests:
+    name: Testing with Node.js ${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version:
+          - '6.x'
+          - '8.x'
+          - '10.x'
+          - '12.x'
+          - '14.x'
+          - '15.x'
```

Comment:
Anything before 12 is unsupported[^1]. I recommend:

```yaml
          - '12.x'
          - '14.x'
          - '16.x'
          - '17.x'
```

[^1]: https://nodejs.org/en/about/releases/

Reply:
Thanks! I've added 16.x and 17.x; we should be able to remove <12.x later.